core/client/src/test/java/alluxio/client/file/FileInStreamTest.java

```diff
@@ -540,23 +535,16 @@ public void skipNegativeTest() throws IOException {
   }
 
   /**
-   * Tests the location policy created with different options.
+   * Tests that the file in stream uses the supplied location policy.
    */
   @Test
-  public void locationPolicyTest() {
-    mTestStream =
-        new FileInStream(mStatus, InStreamOptions.defaults().setReadType(ReadType.CACHE_PROMOTE));
-
-    // by default local first policy used
-    FileWriteLocationPolicy policy = Whitebox.getInternalState(mTestStream, "mLocationPolicy");
-    Assert.assertTrue(policy instanceof LocalFirstPolicy);
-
-    // configure a different policy
-    mTestStream =
-        new FileInStream(mStatus, InStreamOptions.defaults().setReadType(ReadType.CACHE)
-            .setLocationPolicy(new RoundRobinPolicy()));
-    policy = Whitebox.getInternalState(mTestStream, "mLocationPolicy");
-    Assert.assertTrue(policy instanceof RoundRobinPolicy);
+  public void locationPolicyTest() throws Exception {
```

Comment:
Yes, the default configuration functionality belongs to `InStreamOptions`, not `FileInStream`